src/components/UploaderForm.tsx

```diff
@@ -0,0 +1,104 @@
+import React, {
+  FunctionComponent,
+  useState,
+  useMemo,
+  useCallback,
+} from 'react';
+import { Button, ErrorMessage } from 'lbh-frontend-react';
+import { Formik, Form } from 'formik';
+import * as Yup from 'yup';
+import UploaderPanel from './UploaderPanel';
+import { DocumentType } from '../domain/document-type';
+import { EvidenceRequest } from '../domain/evidence-request';
+
+const UploaderForm: FunctionComponent<Props> = (props) => {
+  const [submitError, setSubmitError] = useState(false);
+
+  const requestedDocuments = [props.evidenceRequest.documentType];
```

Comment:
When you merge `main` this will have changed, we now have `documentTypes` as an array to match the API

```suggestion
  const requestedDocuments = props.evidenceRequest.documentTypes;
```